### PR TITLE
Quick Hull implementation in cpp and python

### LINKED ISSUE #736: Quick hull algo in c++ and python
I would like to add these

### PR DESCRIPTION
**Pull request by suyashgautam**
Fixes #736 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Languages Used:
- Cpp
- Python

#### Files Added:
- Quick_Hull.py
- Quick_Hull.cpp

Thanks!
@jainaman224

### CODE REVIEW

**Comment by suyashgautam** [comment]
@singh-shreya6 @raghav-dalmia @asha15

**Comment by jainaman224** [comment]
@suyashgautam From next implement it in only one language.

**Comment by suyashgautam** [comment]
@jainaman224 Okay so is it correct or do i need to do it again?

File: Quick_Hull/Quick_hull.cpp [hunk 1]
```diff
@@ -0,0 +1,132 @@
+/*
+Quickhull Algorithm for Convex Hull
+
+Given a set of points, a Convex hull is the smallest convex polygon containing all the given points.
```
Comment (SakshiShreya) [inline Quick_Hull/Quick_hull.cpp]: Please break your lines. One line should not have more than 80 characters.

File: Quick_Hull/Quick_hull.cpp [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+#define ptspr pair<int,int>
+
+set<ptspr> ipts;
+set<ptspr> finalpts;
+
+
+int findside(ptspr p1,ptspr p2,ptspr p)
+{
+    int val=(p2.first-p1.first)*(p.second-p1.second)-(p2.second-p1.second)*(p.first-p1.first);
```
Comment (SakshiShreya) [inline Quick_Hull/Quick_hull.cpp]: - Break this line. Should not contain more than 80 characters.
- Add space around operators.

File: Quick_Hull/Quick_hull.cpp [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        return 0;
+    }
+
+    if(val<0)
+    {
+        return -1;
+    }
+}
+
+int distance(ptspr p1,ptspr p2,ptspr p)
```
Comment (SakshiShreya) [inline Quick_Hull/Quick_hull.cpp]: Add space after comma.

File: Quick_Hull/Quick_hull.cpp [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+    for(int i=0;i<n;i++)
+    {
+        if(minx>a[i].first)
+            minx=a[i].first;
+
+        if(maxx<a[i].first)
+            maxx=a[i].first;
+    }
+
```
Comment (SakshiShreya) [inline Quick_Hull/Quick_hull.cpp]: Remove extra new-lines.

File: Quick_Hull/Quick_hull.cpp [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+}
+
+int main()
+{
+    int x,y,n;
+    cout<<"Enter number of points \n";
+    cin>>n;
+
+    ptspr a[n];
+    for(int i=0;i<n;i++)
```
Comment (SakshiShreya) [inline Quick_Hull/Quick_hull.cpp]: - Add space after semi-colon

**Comment by suyashgautam** [comment]
I have made the changes can you please review the changes @SakshiShreya

**Pull request merged**